code/modules/admin/verbs/anonymousnames.dm

```diff
@@ -35,24 +25,68 @@
 	var/result = input(usr, "Choose an anonymous theme","going dark") as null|anything in input_list
 	if(!usr || !result || result == "Cancel")
 		return
-	result = input_list[result]
+	var/datum/anonymous_theme/chosen_theme = input_list[result]
+	var/extras_enabled = "No"
 	var/alert_players = "No"
 	if(SSticker.current_state > GAME_STATE_PREGAME) //before anonnames is done, for asking a sleep
+		if(initial(chosen_theme.extras_enabled))
+			extras_enabled = tgui_alert(usr, extras_enabled, "2016 admins didn't miss roundstart", list("Yes", "No"))
```

Comment:
?

Reply:
I have literally no idea what this means

Reply:
hmm... something needs to be rewritten here